src/test/java/ch/liquidmind/inflection/test/blackbox/BlackboxTest.java

```diff
@@ -0,0 +1,101 @@
+package ch.liquidmind.inflection.test.blackbox;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import ch.liquidmind.inflection.compiler.util.InflectionCompilerTestUtility;
+import ch.liquidmind.inflection.model.external.Taxonomy;
+import ch.liquidmind.inflection.model.external.View;
+import ch.liquidmind.inflection.model.external.util.TaxonomyTestUtility;
+import ch.liquidmind.inflection.proxy.Proxy;
+import ch.liquidmind.inflection.proxy.ProxyRegistry;
+import ch.liquidmind.inflection.proxy.util.ProxyGeneratorTestUtility;
+import ch.liquidmind.inflection.test.TestUtility;
+import ch.liquidmind.inflection.test.model.B1;
+import ch.liquidmind.inflection.test.model.TestEnum;
+
+public class BlackboxTest
```

Comment:
I will create tests looking at the compiled/linked taxonomy directly as discussed, but I do not think the ProxyGeneratorTestUtility is affected. The loadProxy() method is for loading and instantiating proxy classes only. For instantiation, a context taxonomy loader must be set, otherwise it will not work. 
Utility methods for taxonomy loading are located in TaxonomyTestUtility.